gocourse06/generalSmartZoo/go.mod

```
@@ -0,0 +1,3 @@
+module generalSmartZoo
```

```suggestion
module github.com/sarff/prjctr-golang_beginning/gocourse06/smartzoo
```

https://github.com/sarff/prjctr-golang_beginning/pull/2#discussion_r1720995037